main/introduction.bs

```diff
@@ -18,6 +18,12 @@ with mechanisms for controlling who can access what.
 A <dfn export>Solid app</dfn> is an application
 that reads or writes data from one or more [=data pods=].
 
+A <dfn export>read operation</dfn> entails that information can be revealed about the content of a resource, and categorically excludes operations to "write", as well as unintended exposure or side-effects within that. [[Source](https://github.com/solid/specification/issues/149#issue-568433265)]
+
+A <dfn export>write operation</dfn> entails an update in that information can be created or removed, and categorically excludes operations to "read", as well as unintended exposure or side-effects within that. A write operation affects a set of resources atomically and recursively. [[Source](https://github.com/solid/specification/issues/126#issuecomment-569920473)]
```

Comment:
"set of" and "recursively" seem to be mutually exclusive. Either it is one resource, and child resources are affected recursively, or we define a set of resource (possibly using recursive means) and the operation affects all members in the set.

Reply:
What do you think of this revision: https://github.com/solid/specification/pull/157/commits/8c28e8edd6bb6e19f21200b870736cd344a095be

Reply:
I see no issues; curious if we need to handle recursion elsewhere now (but not a blocker).

Reply:
Figured generalised description may not suffice or particularly useful here. It may be best to describe recursion in context of something with the interdepencies.